setup/native-script.ps1

```diff
@@ -105,6 +105,20 @@ echo y | cmd /c "$androidExecutable" update sdk --filter "android-23" --all --no
 echo y | cmd /c "$androidExecutable" update sdk --filter "build-tools-23.0.2" --all --no-ui
 echo y | cmd /c "$androidExecutable" update sdk --filter "extra-android-m2repository" --all --no-ui
 
+if ((Read-Host "Do you want to install Android emulator?") -eq 'y') {
```

Comment:
You can use the `Install` function here instead of asking the user explicitly.
The purpose of the `Install` function is that the user may specify that that everything should be installed with no further questions.